packages/botonic-plugin-flow-builder/src/content-fields/flow-knowledge-base.tsx

```diff
@@ -4,19 +4,28 @@ import React from 'react'
 import { ContentFieldsBase } from './content-fields-base'
 import { HtKnowledgeBaseNode } from './hubtype-fields'
 
+export const DEFAULT_MEMORY_LENGTH = 1
+
```

Comment:
maybe `DISABLED_MEMORY_LENGHT`? in flow builder we also have `DEFAULT_MEMORY_LENGTH` and it is set to 10 so this could be confusing